frontend/docs/pages/home/child-spawning.mdx

```diff
@@ -1 +1,416 @@
-    TODO V1 DOCS
+import { Callout, Card, Cards, Steps, Tabs } from "nextra/components";
+import UniversalTabs from "@/components/UniversalTabs";
+import { GithubSnippet, getSnippets } from "@/components/code";
+
+# Procedural Child Workflow Spawning
+
+Hatchet supports the dynamic creation of child workflows during a parent workflow's execution. This powerful feature enables:
+
+- **Complex, reusable workflow hierarchies** - Break down complex workflows into simpler, reusable components
+- **Fan-out parallelism** - Scale out to multiple parallel tasks dynamically
+- **Dynamic workflow behavior** - Create loops and conditional branches at runtime
+- **Agent-based workflows** - Support AI agents that can create new workflows based on analysis results or loop until a condition is met
+
+## Creating Parent and Child Workflows
+
+To implement child workflow spawning, you first need to create both parent and child workflow definitions.
+
+<UniversalTabs items={["Python", "Typescript", "Go"]}>
+<Tabs.Tab title="Python">
+
+```python
+from hatchet_sdk import Context, Hatchet
+from pydantic import BaseModel
+
+hatchet = Hatchet(debug=True)
+
+# Child workflow definition
+class ChildInput(BaseModel):
+    n: int
+
+class ValueOutput(BaseModel):
+    value: int
+
+child = hatchet.workflow(name="child")
+
+@child.task()
+def value_task(input: ChildInput, ctx: Context) -> ValueOutput:
+    return ValueOutput(value=input.n)
+
+# Parent workflow definition
+class ParentInput(BaseModel):
+    n: int
+
+class SumOutput(BaseModel):
+    result: int
+
+parent = hatchet.workflow(name="parent")
+
+@parent.task()
+def sum_task(input: ParentInput, ctx: Context) -> SumOutput:
+    sum_value = 0
+
+    # Run multiple child workflows concurrently
+    child_results = []
+    for i in range(input.n):
+        # Spawn and run a child workflow
+        child_result = child.run_as_child(ctx, ChildInput(n=i))
```

Comment:
this one is incorrect - it should just be `run_no_wait` (or `run` if they want to block) - `run_as_child` doesn't exist